MFVNeutralino/python/AnalysisCuts_cfi.py

```diff
@@ -10,8 +10,8 @@
 
 mfvAnalysisCuts = cms.EDFilter('MFVAnalysisCuts',
                                mevent_src = cms.InputTag('mfvEvent'),
-                               #apply_presel = cms.int32(4) if use_btag_triggers else cms.int32(1),  # 1 = jets, 2 = el/mu, 3 = jets OR bjet/displaced dijet triggers, 4 = bjet/displaced dijet triggers veto HT trigger
                                apply_presel = apply_presel,  # 1 = jets, 2 = el/mu, 3 = jets OR bjet/displaced dijet triggers, 4 = bjet/displaced dijet triggers veto HT trigger, 5 = MET trigger
+                               require_met_filters = cms.bool(True),
```

Comment:
Should we only have this set to true when the MET presel is being used? Or we could nest
```    if (require_met_filters && (!mevent->pass_metfilters))
      return false;
```
in AnalysisCuts inside of the MET trigger's presel.